src/transactions/feed/queryHelper.ts

```diff
@@ -101,7 +101,6 @@ export function useFetchTransactions(): QueryHookResult {
   })
 
   const [fetchingMoreTransactions, setFetchingMoreTransactions] = useState(false)
-  let hasNewTransaction = false
```

Comment:
this variable shouldn't be global, we should decide if each individual response from blockchain-api has new transactions that need to be cached. so moved to the relevant scope